package.json

```diff
@@ -187,5 +189,8 @@
         }
       ]
     ]
+  },
+  "volta": {
+    "node": "16.13.1"
```

Comment:
I think the other repos have node pinned at 16.13.2 right now, if you want bump this

Reply:
good catch. it looks like author and viewer is pinned to 16.14.2. Changed in develop branch too as that is where this file is from